jupyter-extension/MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include LICENSE
 include *.md
 include pyproject.toml
 recursive-include jupyter-config *.json
+recursive-include jupyterlab_pachyderm *.ipynb
```

Comment:
May want to remove in future (this is included for testing) - Need to look test which checks that all files aren't included